core-common/src/main/java/org/zalando/nakadi/annotations/validation/LabelValue.java

```diff
@@ -0,0 +1,25 @@
+package org.zalando.nakadi.annotations.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({
+        ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.ANNOTATION_TYPE,
+        ElementType.TYPE_USE})
+@Retention(RetentionPolicy.RUNTIME)
+@Size(max = 1000)
```

Comment:
```suggestion
@Size(max = 63)
```

From [documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/):

> Valid label value:
> must be 63 characters or less (can be empty),
> unless empty, must begin and end with an alphanumeric character ([a-z0-9A-Z]),
> could contain dashes (-), underscores (_), dots (.), and alphanumerics between.